packages/knip/fixtures/plugins/github-actions/.github/actions/composite/action.yml

```diff
@@ -0,0 +1,135 @@
+name: Action Composite
+description: Description
+
+runs:
+  using: 'composite'
+  steps:
+    - run: asdf install '${{ matrix.tool }}' '${{ steps.version.outputs.latest }}'
+      shell: /bin/bash
```

Comment:
Is it necessary to duplicate all of the existing `test.yml` file here? Could a few steps do to assert the functionality properly?

Reply:
Yup, makes sense - https://github.com/webpro/knip/pull/432/commits/6493adb7ca306e5f40a4e4788c5a3ae064cb1d1a